### ISSUE TITLE
accessibility 

### ISSUE THREAD

**Issue by bantek89** [FAC-Sixteen/week-3-agony-yaunt#24]
accessibility audit passed with flying colours - 100% BUT unable to use a screen reader to navigate site an unable to use navigate with tabbing

great job on labels!

**Comment by bobbysebolao** [comment]
Great point!
